.github/workflows/integrationtests.yaml

```diff
@@ -51,7 +45,14 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: '11'
-          cache: 'gradle'
+
+      - name: Setup Gradle cache
+        uses: gradle/gradle-build-action@v2
+
+      - name: Launch Cosmos DB Emulator
+        run: |
+          Import-Module "$env:ProgramFiles\Azure Cosmos DB Emulator\PSModules\Microsoft.Azure.CosmosDB.Emulator"
```

Comment:
out of curiosity: does this emulator comes together with windows-2019? If not how is it installed?

Reply:
Added comment